content-services/content-sources-backend
# Fixes 2264: remove kafka based inspection

## Summary
This removes the kafka based introspection and workers.  Some kafka code is still present to service notifications.

## Testing steps

To see that notification aren't broken run:
```
make kafka-topic-consume KAFKA_TOPIC=platform.notifications.ingress
```

then create some repository.  You should see a couple messages come through
